packages/itwin/ec3-widget/src/components/EC3/EC3Config.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+export class EC3Config {
+  public CLIENT_ID: string;
```

Comment:
As I understand this class was added mainly for handling some defaulst (default uri, default scopes). Why did it have to change field names?

Change field names to camelCase and make them `readonly`.